docs/configuration.md

```diff
@@ -326,3 +326,20 @@ This parameter configures the minimum zoom level available. Set it to a not-null
 _Default: Infinity_
 
 This parameter configures the maximum zoom level available. Set it to a lower value to prevent your users from zooming in too deeply.
+
+### numberDisplayedTicks
+
+\_Default:
+
+```js
+const chart = eventDrops({
+   numberDisplayedTicks: {
+        small: 3,
+        medium: 5,
+        large: 7,
+        extra: 12,
+    },
+});
+
+This parameter configures the number of ticks display in the header
```

Comment:
I don't understand this sentence. I propose:

> When reducing chart width, we need to display less labels on the horizontal axis to keep a readable chart. This parameter aims to solve the issue. Hence, on smallest devices, it displays only 3 labels by default at the same time. 

And add a list of breakpoint resolutions to know what `small` means.